torch/CMakeLists.txt

```diff
@@ -359,7 +365,9 @@ endif()
 
 if(BUILD_ONEDNN_GRAPH)
   target_compile_definitions(torch_python PRIVATE "-DBUILD_ONEDNN_GRAPH")
-  target_compile_definitions(torch_cpu PRIVATE "-DBUILD_ONEDNN_GRAPH")
+  if(NOT BUILD_LIBTORCHLESS)
+    target_compile_definitions(torch_cpu PRIVATE "-DBUILD_ONEDNN_GRAPH")
```

Comment:
Is this failing with ``BUILD_LIBTORCHLESS`` on ? What is the failure we might want to followup on this